benchmarking/frameworks/framework_base.py

```diff
@@ -162,6 +192,17 @@ def runOnPlatform(self, total_num, cmd, platform, platform_args,
                       converter):
         assert False, "Child class need to implement runOnPlatform"
 
+    def runOnHost(self, cmd):
```

Comment:
Can we just call processRun directly? This way we don't need to create yet another function.

Reply:
mainly want to reuse it for post-processing.